### PR TITLE
Fix issue #1845, try 2

### PR DESCRIPTION
**Pull request by J-N-K**
Clean Commit. Old PR #2445 was messed up while rebasing.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java [hunk 1]
```diff
@@ -525,7 +528,7 @@ private void handleEventType(MonitorEvent event, String bindingType) {
 						.getItemNamesForType(bindingType)) {
 					Class<? extends Item> itemType = provider
 							.getItemType(itemName);
-
+					//logger.info("Processing {} {}", itemName, itemType);
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java]: please uncomment und decrease level to DEBUG or TRACE

File: bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java [hunk 2]
```diff
@@ -473,9 +477,8 @@ private MonitorEvent parseMonitorEvent(String line) {
 			event.timestamp = sections[0];
 			event.eventType = sections[1];
 			event.connectionId = sections[2];
-
 			if (event.eventType.equals("RING")) {
-				event.externalNo = sections[3];
+				event.externalNo = sections[3]; //sections[3].equals("") ? " " : sections[3];
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java]: please remove commented code

**Comment by teichsta** [comment]
Hi @J-N-K thanks for the clean-up. Please find my review comments inline. Best, Thomas E.-E.

File: bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java (hunk 1 above)
Comment (J-N-K) [inline bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java]: This should be removed completely. It was not included in my original commit.

Comment (teichsta) [inline bundles/binding/org.openhab.binding.fritzbox/src/main/java/org/openhab/binding/fritzbox/internal/FritzboxBinding.java]: fine with me as well …

**Pull request merged**